### PR TITLE
feat(gnodev): allow gnodev to load txs

### PR DESCRIPTION
**Pull request by deelawn**
This is useful for testing an app; it allows gnodev to be started each time with a particular state -- the result of the provided transactions.

<details><summary>Contributors' checklist...</summary>

- [ ] Added new tests, or not needed, or not feasible
- [ ] Provided an example (e.g. screenshot) to aid review or the PR is self-explanatory
- [ ] Updated the official documentation or not needed
- [ ] No breaking changes were made, or a `BREAKING CHANGE: xxx` message was included in the description
- [ ] Added references to related issues and PRs
- [ ] Provided any useful hints for running manual tests
- [ ] Added new benchmarks to [generated graphs](https://gnoland.github.io/benchmarks), if any. More info [here](https://github.com/gnolang/gno/blob/master/.benchmarks/README.md).
</details>

### CODE REVIEW

File: tm2/pkg/std/tx.go [hunk 1]
```diff
@@ -136,3 +144,36 @@ func (fee Fee) Bytes() []byte {
 	}
 	return bz
 }
+
+func LoadTxs(ctx context.Context, reader io.Reader) ([]Tx, error) {
```
Comment (gfanton) [inline tm2/pkg/std/tx.go:148]: same here, I think `ParseTxs` should be a better name

File: contribs/gnodev/cmd/gnodev/txs.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/gnolang/gno/tm2/pkg/std"
+)
+
+func loadTxs(txFile string) ([]std.Tx, error) {
```
Comment (gfanton) [inline contribs/gnodev/cmd/gnodev/txs.go:11]: I think `ParseTxs` should be a better name here?

File: contribs/gnodev/cmd/gnodev/main.go [hunk 3]
```diff
@@ -233,10 +241,16 @@ func execDev(cfg *devCfg, args []string, io commands.IO) (err error) {
 	}
 	logger.Debug("balances loaded", "list", balances.List())
 
+	// Load transactions.
+	txs, err := loadTxs(cfg.txsFile)
+	if err != nil {
+		return fmt.Errorf("unable to load transactions: %w", err)
```
[the remark this answers is not in the document]
Comment (deelawn) [reply contribs/gnodev/cmd/gnodev/main.go:247]: done

File: tm2/pkg/std/tx.go (hunk 1 above)
Comment (deelawn) [reply tm2/pkg/std/tx.go:148]: done

File: contribs/gnodev/cmd/gnodev/txs.go (hunk 2 above)
Comment (deelawn) [reply contribs/gnodev/cmd/gnodev/txs.go:11]: done

**Review by gfanton — APPROVED** [review]

**Pull request merged**